web/src/lib/i18n/en.json

```diff
@@ -877,8 +960,16 @@
   "updated_password": "Updated password",
   "upload": "Upload",
   "upload_concurrency": "Upload concurrency",
+  "upload_errors": "Upload completed with {count, plural, one {# error} other {# errors}}, refresh the page to see new upload assets.",
+  "upload_progress": "Remaining {remaining} - Processed {processed}/{total}",
+  "upload_skipped_duplicates": "Skipped {count, plural, one {duplicate} other {duplicates}}",
```

Comment:
Missing the count value and the message changed. Original message is `Skipped ${$duplicateCounter} duplicate asset${s($duplicateCounter)}`